### ISSUE TITLE
Dropdown can be displayed behind of another filed in the row

### ISSUE THREAD

**Issue by martinpesout** [pcbowers/sanity-plugin-tags#13]
I'm playing with this plugin. And I found 2 problems: 

1. When having tag field and field with the rich text editor I'm getting to the problem that tag dropdown can be displayed behind of rich text editor. The user experience when interacting with tag field isn't great in this situation. Is it possible to fix styling of dropdown?
![Snímek obrazovky 2022-08-10 v 11 14 02](https://user-images.githubusercontent.com/239468/183864521-80708131-f79d-4a3c-92f2-9d4fd83a0fb9.png)

2. When having tag field in my schema of some document, it seems that changes aren't highlighted. Changes are highlighted for all other fields in my document excepts fields with `type="tags"`. 
![Snímek obrazovky 2022-08-10 v 11 19 23](https://user-images.githubusercontent.com/239468/183865580-47dd49e6-b39f-4216-98bb-6daa62bf1471.png)

Is it possible to fix these problems? Thank you.

**Comment by JohnGemstone** [comment]
Just came across this overlapping problem, to fix you could add some custom css into your studio like so:
```css
div[class^="TagsInput-module_menu"] {
  z-index: 5!important;
}
```

**Comment by pcbowers** [comment]
Hmm, I'm not a huge fan of adding CSS to a module that currently is CSS free (or at least, is not adding any CSS to what is already included with react tags). Especially when that CSS requires an important flag, meaning it would make it extremely difficult for another user to edit. I think this may be a problem that should be taken up with react-tags or fixed by the user if they have that issue.

That said, open to other suggestions. Will continue to look into it.

**Comment by JohnGemstone** [comment]
Hey @pcbowers! Yeah this was more of a workaround for users rather than a suggestion for the plugin itself.

**Comment by pcbowers** [comment]
@JohnGemstone Thanks for providing it!

**Comment by pcbowers** [comment]
Fixed with #35
